backend/src/main/java/heartbeat/service/report/CSVFileGenerator.java

```diff
@@ -110,6 +132,7 @@ public void convertPipelineDataToCSV(List<PipelineCSVInfo> leadTimeData, String
 
 	public InputStreamResource getDataFromCSV(String dataType, long csvTimeStamp) {
```

Comment:
csvTuneStamp is not readable for anyone, can u help to refactor it with a readable value, like `2023-02-02-14-14-15`

Reply:
This `csvTuneStamp` is only used for naming the CSV files stored on the server. The filename for the CSV files that our users download is defined on the frontend, and the timestamp has already been formatted as 'HHmmSSS', such as metric-20231009-20231020-1947776.csv. Would you think this format is acceptable?

Reply:
No, storing it as a readable name is a basic requirement everywhere, coc, the code and the output are read/identified by a human, not the robot

Reply:
You're right, I've learned. So, do you think it's better for us to format this timestamp in a GET request on the frontend or to format it on the backend?

Reply:
You're right, I've learned. So, do you think it's better for us to format this timestamp on the frontend or to format it on the backend?